datalad_container/containers_add.py

```diff
@@ -321,6 +339,25 @@ def __call__(name, url=None, dataset=None, call_fmt=None, image=None,
                 "{}.cmdexec".format(cfgbasevar),
                 call_fmt,
                 force=True)
+        # --extra-input sanity check
+        # TODO: might also want to do that for --call-fmt above?
+        extra_input_placeholders = dict(img_dirpath="", img_dspath="")
+        for xi in (extra_input or []):
+            try:
+                xi.format(**extra_input_placeholders)
+            except KeyError as exc:
+                yield get_status_dict(
+                    action="containers_add", ds=ds, logger=lgr,
+                    status="error",
+                    message=("--extra-input %r contains unknown placeholder %s. "
```

Comment:
@mih @adswa - I forgot but remember we had some conversation: do we have some syntax sugaring to use in `message` of records to differentiate CLI `--option` from Python API function `option`?

Reply:
not that I'm aware of

Reply:
ok, thanks!